### PR TITLE
Added Project Euler problem 20 in python

### PR DESCRIPTION
**Pull request by mansiag**
**Fixes issue:**
N/A

**Changes:**
Added Project Euler problem 20 in python.

### CODE REVIEW

File: code/online_challenges/src/project_euler/problem_020/problem_020.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+factorial = 1
+for i in range(100):
+	factorial *= (i + 1)
+sum = 0
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_020/problem_020.py]: Add space below. Also, don't use sum for a name.

File: code/online_challenges/src/project_euler/problem_020/problem_020.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+while factorial > 0:
+	sum += factorial % 10
+	factorial = factorial // 10
+print(sum)
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_020/problem_020.py]: Add newline before.

File: code/online_challenges/src/project_euler/problem_020/problem_020.py (hunk 1 above)
Comment (mansiag) [reply code/online_challenges/src/project_euler/problem_020/problem_020.py]: Where should I add space?

Comment (mansiag) [reply code/online_challenges/src/project_euler/problem_020/problem_020.py]: @arnavb Can you please suggest changes here?

Comment (arnavb) [reply code/online_challenges/src/project_euler/problem_020/problem_020.py]: By space I meant newline. Don't use sum for a variable name.

Comment (mansiag) [reply code/online_challenges/src/project_euler/problem_020/problem_020.py]: Thanks @arnavb

**Pull request merged**